src/main/cpp/t10_max_local.cpp

```diff
@@ -24,5 +24,24 @@
 using namespace std;
 
 int t10_max_local() {
+	int a;
+	int x;
+	int b = 1;
+	int count = 0;
+	cin >> a;
+	int o = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'o' is assigned a value that is never used.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4734912)